docs/api/supervisor/endpoints.md

```diff
@@ -2083,6 +2083,7 @@ Returns a dict with selected keys from other `/*/info` endpoints.
 | operating_system | string         | The operating system on the host                             |
 | features         | list           | A list ov available features on the host                     |
 | machine          | string         | The machine type                                             |
+| machine_id       | string or null | The machine ID of the underlying operating system            |
```

Comment:
We generally aren't as diligent about typing here with the `or null` tbh. You'll notice right above `machine` just says `string` even though it also can technically be `or null`.

We can try and be more accurate to typing here going forward if you want. But it might require more explanation then we're doing right now. Like as a developer I'd probably want to know when this can be `null` and what to do when it is. Since we're proposing this as the true way to identify installations in addons.